app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/topics.controllers.js");
+
+app.use(express.json())
+
+app.get(`/api/topics`, getTopics);
+
+app.listen(9090, (error) => {
```

Comment:
I'd recommend putting this in it's own file, when you host later in the week your platform will need to be able to run the file in isolation rather than from within the app file.